pkg/network/vmispec/interface_test.go

```diff
@@ -91,41 +91,6 @@ var _ = Describe("VMI network spec", func() {
 		})
 	})
 
-	const iface1, iface2, iface3, iface4, iface5 = "iface1", "iface2", "iface3", "iface4", "iface5"
-
-	Context("pop interface by network", func() {
```

Comment:
Are these scenarios covered at an upper level?

Reply:
I've dropped this change in order to limit the scope of this PR.

Reply:
ping

Reply:
This scenario should be covered by:
https://github.com/kubevirt/kubevirt/blob/a3b5ee1fc5eddd4a31ec29cf6493a7834cbda5ff/pkg/network/setup/netstat_test.go#L625